pages/book/expressions.mdx

```diff
@@ -0,0 +1,199 @@
+# Expressions
+
+import { Callout } from 'nextra/components'
+
+Every operator in Tact forms an expression, but there's much more to uncover as Tact offers a wide range of expressive options to choose from.
+
+<Callout>
+
+  Note, that in Tact all expressions have higher [precedence](/book/operators#precedence) than any given [operator](/book/operators) except for [parentheses](/book/operators#parentheses), which have the highest precedence of all operators and expressions.
```

Comment:
> all expressions have higher precedence than any given operator

I don't really understand what this means. There is only operator precedence that helps resolve parsing issues with ambiguity. For instance, `+` is an operator, but `2 + 2` is an expression.

Reply:
Oh, yeah, you're right. Mixed those concepts accidentally, will remove those lines.